cli/src/main/scala-2.11/coursier/cli/Helper.scala

```diff
@@ -339,30 +319,26 @@ class Helper(
       }).groupBy(_._1).mapValues(_.map(_._2).toSet).toMap
     }
 
-  val baseDependencies = allModuleVersionConfigs.map {
-    case (module, version, configOpt) =>
-      Dependency(
-        module,
-        version,
-        attributes = Attributes("", ""),
-        configuration = configOpt.getOrElse(defaultConfiguration),
```

Comment:
It seems `defaultConfiguration` isn't taken into account in the new code.

(Yes, the original `cli` code has poor test coverage :| )

Reply:
good catch. will correct this and add a test